File: Aerospike.java
package com.codeinventory.aerospike;

import java.util.Random;

import com.aerospike.client.AerospikeClient;
import com.aerospike.client.Bin;
import com.aerospike.client.Key;
import com.aerospike.client.Record;

public class Aerospike {
	
	
	static int NUM_ENTRIES = 1000000;
	static int MAX_VAL = 12000000;

	
	public static void main(String ... args){
		AerospikeClient client = new AerospikeClient("127.0.0.1", 3000);

		long start = System.currentTimeMillis();
		
		for (int i=0;i<NUM_ENTRIES;i++ ){
		Key key = new Key("test", "demo", getRandomInt());
		Bin bin1 = new Bin("bin1", "value1");

		client.put(null, key, bin1);
		}
		//Record record = client.get(null, key);
		
		System.out.println((System.currentTimeMillis() - start)/1000);

		client.close();
	}
	static Random rand = new Random();
	public static int  getRandomInt(){
		
		int randomNum = 0 + rand.nextInt(MAX_VAL + 1);
		return randomNum;
	}

}

File: aerospike.py
#!/usr/bin/env python
import aerospike as aero

config = {
  'hosts': [ ('localhost', 3000) ]
}
NUM_ENTRIES = 1000000
MAX_VAL = 12000000

try:
  client = aero.client(config).connect()
except:
  print "exception connecting to aerospike"
  import sys
  sys.exit(1)

key = ('test', 'demo', 'foo')

try:

	for i in range(0, NUM_ENTRIES):
    		value = random.randint(0, MAX_VAL)
    		client.put(str(i),{'value':str(value)})

except Exception as e:
  import sys


client.close()
